sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/PersonalizerAdminAsyncClient.java

```diff
@@ -0,0 +1,611 @@
+        }
+    }
+
+    /**
+     * Get the Offline Evaluation associated with the Id.
+     * @param evaluationId Id of the Offline Evaluation.
+     * @param context The context to associate with this operation.
+     * @return the Offline Evaluation associated with the Id along with {@link Response} on successful completion of
+     * {@link Mono}.
+     */
+    Mono<Response<PersonalizerEvaluation>> getEvaluationWithResponse(String evaluationId, Context context) {
+        if (CoreUtils.isNullOrEmpty(evaluationId)) {
+            throw logger.logExceptionAsError(new IllegalArgumentException("'evaluationId' is required and cannot be null or empty"));
+        }
+        return service.getEvaluations().getWithResponseAsync(evaluationId, context)
+            .onErrorMap(Transforms::mapToHttpResponseExceptionIfExists)
+            .map(response -> new SimpleResponse<>(response, response.getValue()));
+    }
+
+    /**
+     * Delete the Offline Evaluation associated with the Id.
+     * @param evaluationId Id of the Offline Evaluation to delete.
+     * @return the completion of {@link Mono}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Void> deleteEvaluation(String evaluationId) {
+        return deleteEvaluationWithResponse(evaluationId).flatMap(FluxUtil::toMono);
+    }
+
+    /**
+     * Delete the Offline Evaluation associated with the Id.
+     * @param evaluationId Id of the Offline Evaluation to delete.
+     * @return the {@link Response} on successful completion of {@link Mono}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<Void>> deleteEvaluationWithResponse(String evaluationId) {
+        try {
+            return withContext(context -> deleteEvaluationWithResponse(evaluationId, context));
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    /**
+     * Delete the Offline Evaluation associated with the Id.
+     * @param evaluationId Id of the Offline Evaluation to delete.
+     * @param context The context to associate with this operation.
+     * @return the {@link Response} on successful completion of {@link Mono}.
+     */
+    Mono<Response<Void>> deleteEvaluationWithResponse(String evaluationId, Context context) {
+        if (CoreUtils.isNullOrEmpty(evaluationId)) {
+            throw logger.logExceptionAsError(new IllegalArgumentException("'evaluationId' is required and cannot be null or empty"));
+        }
+        return service.getEvaluations().deleteWithResponseAsync(evaluationId, context)
+            .onErrorMap(Transforms::mapToHttpResponseExceptionIfExists)
+            .map(response -> new SimpleResponse<>(response, null));
+    }
+
+    /**
+     * List of all Offline Evaluations.
+     * @return List Evaluations on successful completion of {@link Mono}.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public Mono<List<PersonalizerEvaluation>> getEvaluations() {
```

Comment:
[Java guidelines](https://azure.github.io/azure-sdk/java_introduction.html#methods-returning-collections-paging) state returning a PagedFlux even for non-paginated APIs to keep consistency within the SDK.